--- gnome-todo-3.26.1/doc/reference/meson.build.docfix	2017-10-01 00:46:25.032986025 +0200
+++ gnome-todo-3.26.1/doc/reference/meson.build	2017-10-01 01:21:20.642918268 +0200
@@ -48,8 +48,8 @@ glib_doc_path = join_paths(glib_prefix,
 gtk_prefix = gtk_dep.get_pkgconfig_variable('prefix')
 gtk_doc_path = join_paths(gtk_prefix, 'share', 'gtk-doc', 'html')
 
-doc_path = join_paths(gnome_todo_datadir, 'gtk-doc', 'html', meson.project_name())
+doc_path = join_paths(gnome_todo_datadir, 'doc', 'gtk-doc', 'html', meson.project_name())
 
 gnome.gtkdoc(
   meson.project_name(),
           main_sgml: meson.project_name() + '-docs.sgml',
