--- gnome-todo-40.0/doc/reference/meson.build.orig	2021-05-04 00:30:03.530776300 +0200
+++ gnome-todo-40.0/doc/reference/meson.build	2021-05-09 18:53:49.935089521 +0200
@@ -3,6 +3,7 @@ subdir('xml')
 private_headers = [
   'gtd-application.h',
   'gtd-arrow-frame.h',
+  'gtd-command-bar.h',
   'gtd-dnd-row.h',
   'gtd-edit-pane.h',
   'gtd-empty-list-widget.h',
@@ -17,12 +18,18 @@ private_headers = [
   'gtd-manager-protected.h',
   'gtd-notification-widget.h',
   'gtd-plugin-manager.h',
+  'gtd-plugin-todo-txt.h',
+  'gtd-plugin-todoist.h',
   'gtd-provider-dialog.h',
   'gtd-provider-popover.h',
   'gtd-provider-row.h',
   'gtd-provider-selector.h',
+  'gtd-provider-todo-txt.h',
+  'gtd-provider-todoist.h',
   'gtd-resources.h',
   'gtd-task-row.h',
+  'gtd-task-todo-txt.h',
+  'gtd-todoist-preferences-panel.h',
   'gtd-types.h'
 ]
 
@@ -45,7 +52,7 @@ glib_doc_path = join_paths(glib_prefix,
 gtk_prefix = gtk_dep.get_pkgconfig_variable('prefix')
 gtk_doc_path = join_paths(gtk_prefix, 'share', 'gtk-doc', 'html')
 
-doc_path = join_paths(gnome_todo_datadir, 'gtk-doc', 'html', meson.project_name())
+doc_path = join_paths(gnome_todo_datadir, 'doc', 'gtk-doc', 'html', meson.project_name())
 
 gnome.gtkdoc(
   meson.project_name(),
