--- gnome-todo-3.22.1/src/Makefile.am.link	2017-01-11 20:07:07.954705935 +0100
+++ gnome-todo-3.22.1/src/Makefile.am	2017-01-11 20:07:36.361956637 +0100
@@ -104,11 +104,11 @@ gnome_todo_LDFLAGS = \
 	$(GNOME_TODO_WARN_LDFLAGS)
 
 gnome_todo_LDADD = \
-	$(GNOME_TODO_LIBS) \
 	$(top_builddir)/plugins/eds/libeds.la \
 	$(top_builddir)/plugins/dark-theme/libdarktheme.la \
 	$(top_builddir)/plugins/scheduled-panel/libscheduledpanel.la \
-	$(top_builddir)/plugins/today-panel/libtodaypanel.la
+	$(top_builddir)/plugins/today-panel/libtodaypanel.la \
+	$(GNOME_TODO_LIBS)
 
 gnome_todo_DEPENDENCIES = \
 	$(top_builddir)/plugins/eds/libeds.la \
